docs/user/02-configuration.md

```diff
@@ -73,15 +73,16 @@ Use the following sample CRs as guidance. Each can be applied immediately when y
 | **publisherService**                                 | string     |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
 | **specHash** (required)                              | integer    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
-| **state** (required)                                 | string     | Can have one of the following values: `Ready`, `Error`, `Processing`, `Warning`. `Ready` state is set when all the resources are deployed successfully and backend is connected. It gets `Warning` state in case backend is not specified, NATS module is not installed or EventMesh secret is missing in the cluster. `Error` state is set when there is an error. `Processing` state is set if recources are being created or changed. See the next section for more details. |
+| **state** (required)                                 | string     | Can have one of the following values: `Ready`, `Error`, `Processing`, `Warning`. `Ready` state is set when all the resources are deployed successfully and backend is connected. It gets `Warning` state if backend is not specified, NATS module is not installed or EventMesh secret is missing in the cluster. `Error` state is set when there is an error not caused by a user. `Processing` state is set when eventing is initiliazed or backend is being switched. See the next section for more details. |
```

Comment:
"See the next section for more details." in this generated table comes from the CR itself, right? Does that sentence make sense in the original file? 

What if we insert the overview table higher up (around line 16) instead of at the bottom of the document. I assume that table is very interesting for the user; maybe more than the generated tables?